code-of-conduct.md

```diff
@@ -0,0 +1,68 @@
+community. **Not reading the Code of Conduct is not an excuse for
+breaking it**
+
+## The Pledge
+
+We as community members, contributors, and team members pledge to
+make our community a harassment-free experience for everyone, no
+matter one's age, culture, ethnicity, gender identity or expression,
+nationality, physical or mental disabilities, neurodivergencies,
+politics, race, religion, sex, sexual orientation, socio-economic
+status or subcultural affiliation.
+
+We pledge to act and interact in ways that contribute to an open,
+welcoming, diverse, inclusive, and healthy community, and we are
+committed to promoting these concepts.
+
+## Standards
+
+We expect community members to act in a positive, authentic, and
+deliberate way. Consideration and respect provides a basis for a
+solid community, and we request our members actions reflect these
+values. We also expect communities to refrain from talking about, or
+encouraging illegal activities, including, but not limited to,
+piracy, substance abuse, and theft.
+
+Please take into account that online communities bring people
+together from many different cultures and backgrounds. It’s
+important to understand that sometimes the combination of cultural
+differences and online interaction can lead to misunderstandings.
+
+Examples of positive behavours include, but are not limited to
+- Demonstrating kindness and consideration towards others
+- Being respectful of differing viewpoints, experiences, and
+opinions, unless they, directly or indirectly, harm a group of
+people
+- Educating people about why certain actions are harmful
+- Apologizing for, accepting responsibility for, and leaning from
+mistakes
+- Focusing on what is best not just for us as individuals, but for
+the overall community
+
+## Enforcement 
+
+Moderators are responsible for clarifying and enforcing our
+standards of acceptable behavior and will take appropriate and fair
+corrective action in response to any behavior that they deem
+inappropriate, threatening, offensive, or harmful.
+
+If a community member engages in behavior violating this Code of
+Conduct, the members of the moderation team may take any action they
+deem appropriate. Actions may range from a simple warning to the
+most drastic: permanent removal from the community, exclusion from
+any interaction, and loss of all rights in the community.
+
+## Attributions
+
+This document is a modified version of the [QuiltMC Code of 
+Conduct](https://quiltmc.org/en/community/code-of-conduct/), which
+itself takes inspiration from the [Contributor Covenant version
+2](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html),
+[JetBrains Open Source and Community Code of
+Conduct](https://confluence.jetbrains.com/display/ALL/JetBrains+Open+Source+and+Community+Code+of+Conduct),
+[Microsoft Open Source Code of Conduct](https://microsoft.github.io/codeofconduct),
+and [Open Source Design Code of Conduct](https://opensourcedesign.net/code-of-conduct).
```

Comment:
```suggestion
This document is a modified version of the
[QuiltMC Code of Conduct](https://quiltmc.org/en/community/code-of-conduct/),
which itself takes inspiration from the
[Contributor Covenant version 2](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html),
[JetBrains Open Source and Community Code of Conduct](https://confluence.jetbrains.com/display/ALL/JetBrains+Open+Source+and+Community+Code+of+Conduct),
[Microsoft Open Source Code of Conduct](https://microsoft.github.io/codeofconduct),
and [Open Source Design Code of Conduct](https://opensourcedesign.net/code-of-conduct).
```
It would also be possible to convert this to reference style links, moving the actual URLs out of line: 
```suggestion
This document is a modified version of the [QuiltMC Code of Conduct][quiltmc],
which itself takes inspiration from the [Contributor Covenant version 2][cco],
[JetBrains Open Source and Community Code of Conduct][jetbrains],
[Microsoft Open Source Code of Conduct][microsoft],
and [Open Source Design Code of Conduct][opensourcedesign].

[quiltmc]: https://quiltmc.org/en/community/code-of-conduct/
[cco]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
[jetbrains]: https://confluence.jetbrains.com/display/ALL/JetBrains+Open+Source+and+Community+Code+of+Conduct
[microsoft]: https://microsoft.github.io/codeofconduct
[opensourcedesign]: https://opensourcedesign.net/code-of-conduct
```